shared/desktop/app/app-state.js

```diff
@@ -300,7 +300,15 @@ export default class AppState {
     }
     let displayBounds = screen.getDisplayMatching(rect).bounds
     console.log('Check bounds:', rect, state.displayBounds, displayBounds)
-    return isEqual(state.displayBounds, displayBounds)
+    return (
```

Comment:
Perhaps update the comment at the top of the function to say we're also checking that the window isn't off the screen, and maybe add a link to the electron issue?